backuprestore.adoc

```diff
@@ -1553,14 +1553,80 @@ auth tokens generated prior to the backup will no longer work.
 The Jira ticket ZMS-614 has been opened to track the potential future development
 of ephemeral data backups.
 
-=== Backups with SSDB Backend
+=== SSDB Backend
```

Comment:
Maybe *Ephemeral store SSDB backend*

Reply:
Isn't this clear enough considering it's under the heading of `Notes on Ephemeral Data`?

Reply:
I can prefix it if you like however.

Reply:
Not hugely fussed.